smart_contracts/contracts/Web3Forms.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier:MIT
+
+pragma solidity ^0.8.4;
+
+contract Web3Forms {
+
+    uint formsCount;
+    address public s_owner;
+
+    modifier onlyOwner{
+        require(msg.sender == s_owner);
+        _;
+    }
+
+    constructor(){
+        s_owner= msg.sender;
+    }
+
+    struct Form{
+        uint formId;
+        address creator;
+        uint publishedTime;
+        string formData;
```

Comment:
For now we are assuming the form data to be in string?

Reply:
Yes so for now this `string formData` is representing the IPFS hash which will contain all the data in JSON format

Reply:
Cool, That makes sense ✅